jscomp/others/js_dict.ml

```diff
@@ -27,6 +27,7 @@ type 'a t
 type key = string
 
 external get : 'a t -> key -> 'a option = "" [@@bs.get_index] [@@bs.return {undefined_to_opt}]
+external unsafe_get : 'a t -> key -> 'a = "" [@@bs.get_index] 
```

Comment:
I'm a bit iffy about this. This isn't really an "unsafe" function, it's just incorrect. While it's certainly not safe, "unsafe" to me means it will throw, not ignore errors and pass on the hot potatoes. Such functions should not exist at all IMO, and I think the correct thing to do here is to define a normal function that uses `get`, but raises an exception on `None`.